src/utils/internal/parseGraphQLRequest.ts

```diff
@@ -3,6 +3,7 @@ import type {
   OperationDefinitionNode,
   OperationTypeNode,
 } from 'graphql'
+import { parse } from 'graphql'
```

Comment:
While we're at it, I believe there's a place or two where we import types from `graphql` but do not mark those imports as `import type`. Could you please look into those?

Reply:
Audited those here (mostly in tests) - 1dbc669ce1b9d2bfb229bc655542c336098804f9

Reply:
https://github.com/mswjs/msw/blob/main/src/handlers/GraphQLHandler.test.ts#L5

Just calling this out separately.  the import for `OperationTypeNode` is used as an enum in this test, so we can't safely just import types there

Reply:
Oh, great catch! We need to be careful with enums. Luckily, those won't compile if we import them incorrectly.